requirements.txt

```diff
@@ -9,7 +9,7 @@ matplotlib
 # we'd prefer not to burden users with another prerequisite. 
 # NB: It's possible this fix gets included upstream in a future skimage release, see:
 #     https://github.com/scikit-image/scikit-image/issues/4780#issuecomment-824678872
-msvc-runtime; sys.platform == 'win32'
+msvc-runtime==14.29.30133; sys.platform == 'win32'
```

Comment:
> but it's nice being able to rely on the fact that requirements-freeze.txt includes a specific version for each package listed.

That was my thinking, too! 

It felt odd to have every package have a strictly pinned version #, yet have this one library stay unpinned. Not that I would expect anything to break because of it, but having a strictly pinned version would give me peace of mind for SCT's stable releases. :)